addons/fluxcd/definitions/helm-release-def.yaml

```diff
@@ -106,12 +106,12 @@ spec:
         }
 
         parameter: {
-        	repoType: *"helm" | "git" | "oss"
```

Comment:
we should keep the oss args here and only hide in UX

Reply:
Yes, I'm making sure if we can hide an option of a parameter in uiSchema with @barnettZQG.

Reply:
```
- jsonKey: repoType
    sort: 12
    description: You can set the value of volumeMode to Block to use a volume as a raw block device.
    uiType: Select
    validate:
      defaultValue: helm
      options: 
        - label: Helm Repo
          value: helm
        - label: Git Repo
          value: git
```

Reply:
fixed